.github/workflows/plugin-ci.yml

```diff
@@ -96,11 +130,18 @@ jobs:
           path: dist
 
       - name: ci/prepare-artifact
-        run: mv dist/*.tar.gz dist/$GITHUB_REPOSITORY-ci.tar.gz
+        run: mv *.tar.gz ${GITHUB_REPOSITORY#*/}-ci.tar.gz
+        working-directory: dist
 
-      - name: ci/upload-artifact
-        uses: mattermost/actions/artifact-upload@a137067c750390bfc228375dbe745b2aa3a566e8
+      - name: ci/aws-configure
+        uses: aws-actions/configure-aws-credentials@67fbcbb121271f7775d2e7715933280b06314838 # v1.7.0
         with:
-          artifacts: dist
-          bucket: mattermost-plugins-ci/ci/
-          region: us-east-2
+          aws-region: ${{ inputs.region }}
+          aws-access-key-id: ${{ secrets.aws-access-key-id }}
+          aws-secret-access-key: ${{ secrets.aws-secret-access-key }}
+
+      - name: ci/artifact-upload
```

Comment:
Since we define this job as `delivery`, should we rename this to `cd/artifact-upload` ?